renderer.php

```diff
@@ -58,12 +58,12 @@ public function formulation_and_controls(question_attempt $qa,
             $inputvalue = $qa->get_last_qt_var($fieldname);
             $number = $key + 1;
             $feedback = '';
-            if ($options->feedback) {
+            if ($options->generalfeedback) {
                 $feedback = $question->format_text($answer->feedback, $answer->feedbackformat,
                     $qa, 'question', 'feedback', $answer->answerid);
             }
             $clue = $question->format_text($answer->clue, $answer->clueformat, $qa, 'question', 'clue', $answer->answerid);
-            $title = get_string(
+            $label = format_string(get_string(
```

Comment:
Hi Tim,
I want to remove unnecessary HTML so the accessibility screen reader tool to read the clue correctly.
Example: When we insert the clue with the editor:
It will include <p></p>
1 Across. <p>where is the Christ the Redeemer statue located in<p>?, 6 letter word" to "BRAZIL
so I use format_string to strip the tags.
Is that clear, I did try to use strip_tags in the previous commit but we have format_string to format the string plain text, so I use it.

Reply:
That is not what format_string is for. Are you trying to do better done by using 'clue' => html_to_text($clue, 0, false), in the array below?

Reply:
Oh! I see you almost got this fixed in the second commit. html_to_text is better than strip_tags. E.g. if there is an impage, it replaces the image with the alt text.